distributed/src/main/scala/asobu/distributed/protocol/DRequest.scala

```diff
@@ -0,0 +1,65 @@
+package asobu.distributed.protocol
+
+import akka.stream.Materializer
+import akka.util.ByteString
+import asobu.distributed.Headers
+
+import play.api.http.HttpEntity
+import play.api.mvc._
+
+import scala.concurrent.Future
+
+@SerialVersionUID(1L)
+case class HttpStatus(code: Int) extends AnyVal
```

Comment:
We are pulling Play! into each service now, not sure if that was the case before.  However, since we are using Play! in both front and back end..does it make sense to just use their classes wholesale, and not provide wrappers?